src/west/commands/command.py

```diff
@@ -45,7 +45,8 @@ def __init__(self, **kwargs):
         super(ExtensionCommandError, self).__init__(**kwargs)
 
 
-_NO_TOPDIR_MSG_FMT = '''Error: "{}" is not in a west installation.
+_NO_TOPDIR_MSG_FMT = '''\
+no west installation found from "{}"; "west {}" requires one.
```

Comment:
I think we should change the name from `west installation` to something else now that west is not part of the installation